EIPS/eip-1962.md

```diff
@@ -0,0 +1,224 @@
+---
+eip: 1962
+title: EC arithmetic and pairings with runtime definitions
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/1962
+type: Standards Track
+category: Core
+status: Draft
+created: 2019-04-22
+requires: 1109 (recommended)
```

Comment:
This can only be a number. Since you still use the wording "required" below I suggest just to keep this as `requires: 1109` and it can be discussed later.

Reply:
Fixed